src/app/personalised-homepage/page.js

```diff
@@ -83,10 +83,15 @@ function PersonalisedHomepage() {
         return userProduct === "premium"
     }
 
+    // UserProfilePic function checks whether user 
+
+    const UserProfilePic = () => {
+        return userImage !== null
```

Comment:
When the user doesn't have an image, `userImage` becomes an empty string. 

This means that this statement will always return true, regardless of the user having a profile picture or not.

```js
"" !== null // returns true
```